scripts/helper.js

```diff
@@ -660,6 +659,39 @@ HELPER.generate = (() => {
 		},
 
 		/**
+		 * Get all the dependencies and their child dependencies
+		 *
+		 * @param {object} dependencies      - An object containing dependency { name: version }
```

Comment:
A good way to describe the shape of an option in jsdocs is to add the keys like so:
```js
		/**
		 * Get all the dependencies and their child dependencies
		 *
		 * @param {object} dependencies         - An object containing dependency
		 * @param {object} dependencies.[name]  - The version string with the name as the key
		 * @param {object} dependencyBundle     - An empty object to add the found dependencies to
		 *
		 * @return {array} dependencyBundle  - An object containing all of the dependencies found
		 */
```

Reply:
Will add that now!